admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.js

```diff
@@ -105,6 +105,17 @@ export default {
   productEditLocationText: '.editProductLocation',
   productEditAvailableText: '.editProductAvailable',
   productEditTotalPriceText: '.editProductTotalPrice',
+  // Product Pack Modal
+  productPackModal: '#product-pack-modal',
```

Comment:
You should be able to create another level, really easier to read :sweat_smile: 
```suggestion
  productPackModal: {
    modla: '#product-pack-modal',
    table: '#product-pack-modal-table tbody',
    rows: '#product-pack-modal-table tbody tr:not(#template-pack-table-row)',
    template: '#template-pack-table-row',
    product: {
      img: '.cell-product-img img',
      name: '.cell-product-name .product-name',
      ref: '.cell-product-name .product-reference',
      supplierRef: '.cell-product-name .product-supplier-reference',
      quantity: '.cell-product-quantity',
      availableQuantity: '.cell-product-available-quantity',
    },
  },
```

Reply:
Done. Thanks